services/petstore/pkg/resource/pet/manager.go

```diff
@@ -150,15 +150,16 @@ func (rm *resourceManager) newCreateRequestPayload(
 // observed and decides whether to call the resource manager's Update method
 func (rm *resourceManager) Update(
 	ctx context.Context,
-	res acktypes.AWSResource,
+	resDesired acktypes.AWSResource, // desired
+	resLatest acktypes.AWSResource, // latest
```

Comment:
FYI, we're getting rid of the petstore and bookstore services here:

https://github.com/aws/aws-controllers-k8s/pull/305

So you won't need this after you rebase.

Reply:
Rebased with latest from main.